### PR TITLE
Sitecore Embedded JSS App example

### PR DESCRIPTION
**Pull request by ambrauer**
- Migration of the [sitecore-embedded-jss-app](https://github.com/Sitecore/jss/tree/dev/samples/sitecore-embedded-jss-app) from jss repo
- [Existing docs](https://jss.sitecore.com/docs/techniques/mvc-integration/client-side-embedding) were moved into the Readme
- The old "EmbeddedWizard-items-0.1.zip" was replaced with serialized Sitecore items (via Sitecore CLI)
- Introduced a container environment and "up" script so it can be fired up and fully working "OOTB"

**NOTE**: The JSS npm dependencies in package.json currently target a _prerelease_ version of JSS 19 (19.0.0-canary.63). These need to be updated to final version once 19 is released. The container environment's platform and Headless Services module should be updated to 10.2 and 19.0.0 (respectively) as well.

### CODE REVIEW

File: examples/sitecore-embedded-jss-app/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Sitecore-embedded-jss-app
+
+| | |
+|--|--|
+| **Framework** | JavaScript Rendering SDK |
+| **Sitecore Version** | 10.2 |
```
Comment (CobyPear) [inline examples/sitecore-embedded-jss-app/README.md:6]: Is this right or is it 10.1 still?

File: examples/sitecore-embedded-jss-app/src/rendering/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{
+  "root": true,
+  "extends": [
+    "prettier",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:prettier/recommended",
```
Comment (CobyPear) [inline examples/sitecore-embedded-jss-app/src/rendering/.eslintrc]: This plugin should be last in the array
https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

File: examples/sitecore-embedded-jss-app/src/rendering/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (CobyPear) [inline examples/sitecore-embedded-jss-app/src/rendering/.vscode/settings.json]: These settings seem kind of odd to include, do we need this file?

**Review by CobyPear — COMMENTED** [review]
Looks good to me! Just a few questions.

File: examples/sitecore-embedded-jss-app/README.md (hunk 1 above)
Comment (ambrauer) [reply examples/sitecore-embedded-jss-app/README.md:6]: This is correct (once 10.2 and JSS 19 are public). See "NOTE" in PR description. We will have to come back in and update those dependencies.

File: examples/sitecore-embedded-jss-app/src/rendering/.eslintrc (hunk 2 above)
Comment (ambrauer) [reply examples/sitecore-embedded-jss-app/src/rendering/.eslintrc]: Fixed.

File: examples/sitecore-embedded-jss-app/src/rendering/.vscode/settings.json (hunk 3 above)
Comment (ambrauer) [reply examples/sitecore-embedded-jss-app/src/rendering/.vscode/settings.json]: Yeah, I agree, I've removed this and launch.json.

**Pull request merged**